shapely/algorithms/polylabel.py

```diff
@@ -108,56 +34,9 @@ def polylabel(polygon, tolerance=1.0):
     >>> from shapely import LineString
     >>> polygon = LineString([(0, 0), (50, 200), (100, 100), (20, 50),
     ... (-100, -20), (-150, -200)]).buffer(100)
-    >>> polylabel(polygon, tolerance=10).wkt
-    'POINT (59.35615556364569 121.83919629746435)'
+    >>> polylabel(polygon, tolerance=0.001)
```

Comment:
@jorisvandenbossche has something about tolerance changed in the implementation so that we'd need to tighten the value? Like, does `10` no longer succeed?

Reply:
If I recall correctly, I changed this because 10 was just not really a realistic example. 

The docstring explains `tolerance` as "`tolerance` represents the highest resolution in units of the input geometry that will be considered for a solution."  
So it stops when the search area is smaller than 10 (wide or high, not sure exactly). But so for coordinates in the range of 0-200, that's seems a bit stupid to already stop with a precision of 10

Reply:
(of course, it is is meant for just labeling on a plot, you might not care about a good precision, but I think for the example it is better to set a bit lower tolerance)

Reply:
And if you have lat/lon coordinates, the default of 1.0 is also typically too large, see eg https://github.com/shapely/shapely/issues/1383 where the default caused confusion. For projected coordinates that default is of course typically fine.